game-core/src/main/java/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -2352,58 +2321,13 @@ private void showGame() {
     mapPanel.setRoute(null);
   }
 
-  private void showMapOnly() {
-    // Are we coming from showHistory mode or showGame mode?
```

Comment:
This code seems oddly familiar ^^
I think I created a PR for this, but I don't remember what it was about 😅

Reply:
'showMapOnly' is a copy/paste of the above block, 'showCurrentGame' and also 'showHistory'.

I wonder if that is why it is familiar, or if for other reason.

Reply:
I think I did a refactoring that touched all of the 3 methods.

Found it via git blame: #4737